frontend/libs/studio-components/src/components/StudioAvatar/StudioAvatar.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { type ReactElement } from 'react';
+import classes from './StudioAvatar.module.css';
+import { PersonCircleIcon } from '@studio/icons';
+
+type ImageDetails = {
+  src: string;
+  alt: string;
+  title: string;
+};
```

Comment:
Can we use the native typing for the `<img>` instead? 😊 
```suggestion
type AvatarElement = HTMLAttributes<HTMLImageElement>;
```

Reply:
I didn't think about that, great suggestion! 😄